### DISCUSSION TITLE
Why does Raven Studio uses the canvas ?

### DISCUSSION THREAD

**Discussion by Willienn** [ravendb/ravendb#19212] [Q&A] [answered]
My browser blocks canvas access by default, and I’ve noticed that without it, the Raven favicon appears broken. Why is that?

![image](https://github.com/user-attachments/assets/e5fc9a2c-3526-4f0a-92f3-459dcc564453)
![image](https://github.com/user-attachments/assets/c53a267c-489b-42e9-bc79-443ebfcfdf90)

**Comment by ayende** [comment, marked as answer]
We use canvas extensively in the studio - primarily for graphs, indexing performance, etc.
We also use the `favicon` to indicate what is the node tag. If you have multiple nodes in the cluster, the node tag can tell you which tab you are on

**Reply by Willienn** [reply]
oooh, i really like the ideia of changing the favicon, im gonna look about it
